templates/Datashades-OpsWorks-CKAN-Stack.cfn.yml.j2

```diff
@@ -335,7 +331,7 @@ Resources:
         Statement:
           - Effect: Allow
             Action: "lambda:InvokeFunction"
-            Resource: !Ref InstanceSetupLambda{{ instance_setup_source_hash }}
+            Resource: !GetAtt InstanceSetupLambda{{ instance_setup_source_hash }}.Arn
```

Comment:
this is the only difference, the rest of the changes in this file is a flavour of cfn usage. 
so you missed the .Arn part.